---
title: BEDICT models
summary: 
tags:
#- Featured
#- Clinical Data Science
#- Genome Data Science
#- Translational Bioinformatics
#- Speech & Cognition
date: "2025-11-01T00:00:00Z"
authors: 
- ahmedallam

active: false

# Optional external URL for project (replaces project detail page).
# external_link: ""

image:
  caption: 
  focal_point: Smart

links:
#- icon: github
#  icon_pack: fab
#  name: View Code
#  url: https://github.com/uzh-dqbm-cmi/template

url_code: "https://github.com/uzh-dqbm-cmi/BEDICT-V2"
url_pdf: "https://link.springer.com/content/pdf/10.1186/s13059-025-03586-7.pdf"
url_slides: "https://go.bedict.app/"
url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides: example
---

BEDICT2.0 is a deep learning model that accurately predicts adenine base editing (ABE) efficiency in both lab cells and living models. It was trained on a dataset of 2,195 disease-causing mutations to ensure high accuracy. This tool helps select the best editing strategies for therapeutics while avoiding unwanted changes. (read more here: [link](https://link.springer.com/article/10.1186/s13059-025-03586-7 ))

App: https://go.bedict.app/ 

{{< figure src="bedict_1.png" >}}


BEDICT is the original deep learning algorithm predicting both adenine and cytosine base editing results. It uses data from over 28,000 sequences to account for varying efficiency based on target location. BE-DICT gives reliable predictions that help optimize gene editing strategies. (read more here: [link](https://www.nature.com/articles/s41467-021-25375-z))

Code: https://github.com/uzh-dqbm-cmi/crispr 
App: http://130.60.24.130/page-set?actionID=5f8c494b8c854d0029ffa9d3 

